src/karapace/schema_registry_apis.py

```diff
@@ -206,6 +207,13 @@ def _add_schema_registry_routes(self) -> None:
             schema_request=True,
             auth=self._auth,
         )
+        self.route(
+            "/master_available",
+            callback=self.master_available,
+            # post because they cannot be cached, need to be sure always gathering the real value of that property
+            method="POST",
```

Comment:
Not a big fan of this because it violates semantics of POST. Maybe set a `Cache-Control: no-store, no-cache, must-revalidate` header instead?

Reply:
Agreed, I'll change it